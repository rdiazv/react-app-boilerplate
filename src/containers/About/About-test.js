import {Â React, mount } from 'helpers/test-helper'
import About from 'containers/About'

describe('About', () => {
  it('should render', () => {
    const component = mount(<About />)

    expect(component).toMatchSnapshot()
  })
})
