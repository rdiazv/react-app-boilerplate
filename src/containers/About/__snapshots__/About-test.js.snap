// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render 1`] = `
<About>
  <div>
    <h1>
      About
    </h1>
    <Link
      replace={false}
      to="/"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Back to home
      </a>
    </Link>
  </div>
</About>
`;
